larray_editor/editor.py

```diff
@@ -657,13 +670,87 @@ def load_example(self):
                 self._open_file(filepath)
 
     def open_documentation(self):
-        QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/"))
+        QDesktopServices.openUrl(QUrl(urls['doc_stable']))
 
     def open_tutorial(self):
-        QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/tutorial.html"))
+        QDesktopServices.openUrl(QUrl(urls['doc_tutorial']))
 
     def open_api_documentation(self):
-        QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/api.html"))
+        QDesktopServices.openUrl(QUrl(urls['doc_api']))
+
+    def report_issue(self):
+        if PY2:
+            from urllib import quote
+        else:
+            from urllib.parse import quote
+
+        versions = get_versions()
+        issue_template = """\
```

Comment:
BTW, I did not realize it was possible to use a template on the client side. This is great.

Reply:
You should thank Spyder dev for that.